src/store/post.ts

```diff
@@ -0,0 +1,37 @@
+import { AnyAction } from 'redux';
+
+// Action types
+const SET_POSTS = 'SET_POSTS';
+const REMOVE_POST = 'REMOVE_POST';
+const REMOVE_COMMENT = 'REMOVE_COMMENT';
+
+// Action creators
+
+export const setPosts = (posts: PostFromServer[]) => ({ type: SET_POSTS, posts });
+export const removePost = (postsId: number) => ({ type: REMOVE_POST, postsId });
+export const removeComment = (postsId: number, commentId: number) => (
+  {
+    type: REMOVE_COMMENT,
+    postsId,
+    commentId,
+  });
+
+// posts reducer receives only the `posts.message` part, but not the entire Redux state
```

Comment:
there is no need to keep all those comments, they are needed for you before you started making all those things